apps/sms/views/shared/test/unit/sms_test.js

```diff
@@ -326,6 +326,13 @@ suite('SMS App Unit-Test', function() {
         assertNumberOfElementsInContainerByTag(container, 5, 'input');
       });
 
+      test('Sets up the gaia header for the edit form', function() {
```

Comment:
The selection related tests does not live in the inbox/conversation unit test but sms test for a long time. Adding the tests here for now but we might need to move them back to view specific unit test.